sui/wormhole/sources/governance_message.move

```diff
@@ -281,12 +246,13 @@ module wormhole::governance_message_tests {
         let worm_state = take_state(scenario);
         let the_clock = take_clock(scenario);
 
-        let msg =
-            governance_message::parse_and_verify_vaa_test_only(
+        let parsed =
```

Comment:
nit: the name `parsed` is misleading, because these `VAA`s are not only parsed, but also (sig)verified